content/post/why-should-we-trust-ml.md

```diff
@@ -0,0 +1,74 @@
+
+An open-source package to implement LIME can be found here: https://github.com/marcotcr/lime
+
+### Use Case: Clinical Operations
+
+Consider the use case in which we want to predict which hospitals will enroll more patients. We have many features, and find that what drives hospital enrollment is not very simple, and involves many interactions between variables, as each hospital is different. We build a black-box model that far out performs any linear model in terms of predictive accuracy. Our client is very satisfied with our predictive performance, but wants to understand how we can interpret the predictions of this model, without sacrificing any predictive power. Specifically, what drives the **direction** of the predicted enrollment rate? How can we get personalized explanations? Specifically, why Hospital B gets more enrollment than Hospital A?
+
+By applying LIME, we approximate a linear model for both the predictions of Hospital A and B. We recieve coefficeint estimates for our features around the local regions of these predictions.
+
+![](/img/lime_clinical.png)
+
+We can see that Hospital B has a higher predicted enrollment rate by our black box model and our approximate localized linear model. But why? With LIME, we can see that feature G is what contributes to the difference in enrollment rates. An analysis of that feature and how it differs between the two hospitals would help us understand why these enrollment rates are different, while also giving credit to the other positive features, and understanding that the number of trials in a city actually has a strong negative effect across both hospitals.
+
+We did not have to forego any predictive power to extract the interpretability we needed. It is important to note, however, that LIME gives us *local* interpretations, but if *global* interpretations are needed, some trade-off will need to occur.
+
+### SHAP
+
+SHAP values (short for SHapley Additive exPlanations) provides another framework for interpreting model predictions. It is based on Shapley values, a technique used in game theory to determine how much each player in a collaborative game has contributed to its success. In our case, each SHAP value measures how much each feature in our model contributes, either positively or negatively, to a model output. Its main benefits are that (1) it is completely model agnostic, so can be applied to tree based and black-box models and (2) each individual observation will have its own set of SHAP values, resulting in a local interpretation similar to LIME.
```

Comment:
> tree based and black-box models 

Some tree based models are pretty black boxy! Maybe you mean "and other black-box models"? Or, even better, "any predictive model"(?)